frontend/src/bundles/landing/components/third-step-part/styles.module.scss

```diff
@@ -1,95 +1,104 @@
+.body {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    overflow: hidden;
+    background-color: var(--color-blue-100);
+}
+
 .container {
     display: flex;
     flex-direction: row;
     justify-content: space-between;
     align-items: center;
-    width: 100%;
+    width: var(--container);
     height: 680px;
-    padding: 0 var(--p-8);
-    background-color: var(--color-blue-100);
+    padding: 0 var(--p-3);
 }
 
 .chartCardsContainer {
     position: relative;
-    width: auto;
 }
 
-.chartCardTop {
+.chartCardTop,
+.chartCardBottom {
     position: absolute;
+    display: block;
+    max-width: 420px;
+}
+
+.chartCardTop {
     right: 150px;
     bottom: -20px;
 }
 
 .chartCardBottom {
-    position: absolute;
     top: 0;
     right: -40px;
 }
 
-@media (max-width: 768px) {
-    .container {
-        flex-direction: column;
-        height: auto;
-        padding: var(--p-4) 0;
+@media (max-width: 1200px) {
+    .chartCardTop {
+        right: 100px;
+        bottom: -20px;
     }
 
-    .chartCardsContainer {
-        position: inherit;
-        display: flex;
-        flex-direction: column;
-        gap: 20px;
-    }
+    // .container {
```

Comment:
Remove commented code

Reply:
Resoved in [214d526](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/214d526cd24228787ecabce6767c0fe9ecec563f)